custom/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ plugins.apply(KiwixConfigurationPlugin::class)
 android {
   defaultConfig {
     applicationId = "org.kiwix"
+    ndkVersion  = "21.0.6113669"
```

Comment:
remove this

Reply:
I'll try again after removing this, but the build was failing unless I specifed this.